kyo-core/shared/src/main/scala/kyo/StreamCoreExtensions.scala

```diff
@@ -413,12 +414,9 @@ object StreamCoreExtensions:
                                                 Loop(initialPar): currentPar =>
                                                     if currentPar > 0 then
                                                         parRef.updateAndGet(_ - 1).andThen:
-                                                            // java.lang.System.err.println(s"RUNNING ASYNCHRONOUSLY $input")
```

Comment:
My sophisticated debugging strategy 🤣

Reply:
There's `Debug.values` as well :) it prints with the frame location directly to the console without suspending